qualtran/cirq_interop/t_complexity_protocol.py

```diff
@@ -133,22 +141,22 @@ def _from_bloq_build_call_graph(stc: Any, fail_quietly: bool) -> Optional[TCompl
         return None
     ret = TComplexity()
     for bloq, n in sigma.items():
-        r = t_complexity(bloq, fail_quietly=fail_quietly)
+        r = t_complexity(bloq)
```

Comment:
We need to change the `_, sigma = stc.call_graph(max_depth=1)` line above to the following; so that the `t_complexity` protocol doesn't encounter cirq gates for which an equivalent Bloq exists. 

```
from qualtran.resource_counting.generalizers import cirq_to_bloqs

_, sigma = stc.call_graph(max_depth=1, generalizer=cirq_to_bloqs)
```